filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -329,6 +341,13 @@ You can force Filebeat to close the file as soon as the file name changes by set
 `force_close_files` option to true. The default is false. Turning on this option can lead to loss of data on
 rotated files in case not all lines were read from the rotated file.
 
+[[configuration-global-options]]
+=== Filebeat Global Configuration
+
+You can specify configuration options that control Filebeat behavior at a global level.
+
+==== Options
```

Comment:
@ruflin It's not exactly an asciidoc issue. In most cases, we could remove this heading and move everything up a level, but then we'd get inconsistent formatting for options because in some places we break the options down into further categories. I think it's better to keep all the options at the same heading level so that they appear consistent across the doc (same formatting, etc), and so that you don't get a long list of options under the "On this page" area above the navigation.